coq-mathcomp-algebra-tactics.opam

```diff
@@ -24,7 +24,7 @@ commutative ring. Another key feature of Algebra Tactics is that they
 automatically push down ring morphisms and additive functions to leaves of
 ring/field expressions before applying the proof procedures."""
 
-build: [make "-j%{jobs}%"]
+build: [make "-j%{jobs}%" "build"]
```

Comment:
This is wrong for two reasons:
- This file is auto-generated from meta.yml.
- There is a `run-test` field below. What's written in this `build` field should not run the test suite.

Reply:
Oh, wait. Did you change the behavior of `make` to run the test suite? How does that run the test suite in other CI? (Well, that looks wrong anyway.)

Reply:
> This file is auto-generated from meta.yml.

Right, we have to set the `make_target` variable there apparently.

> How does that run the test suite in the CI of MathComp and Rocq opam archive?

I don't know about the OPAM archive CI but we'll get it into Rocq's own CI and all the nix based one (including MC) which is the most important.

Reply:
Although I agree that we should run the test suite in Rocq and MathComp CI, that should be done on the CI side, not by tweaking the default build target.

Reply:
Unfortunately, doing this requires some non trivial work (otherwise it would already have been done for long). Meanwhile, I prefer having something that "looks wrong" than something potentially broken without us even noticing. Note that the proposed Makefile scheme is not so exotic, at least elpi and HB are doing something similar.

Reply:
I suggest discussing it tomorrow.